testing/regress/ecl-test.json

```diff
@@ -4,7 +4,7 @@
         "espIp" : "127.0.0.1",
         "espSocket": "8010",
         "dropzoneIp" : "127.0.0.1",
-        "dropzonePath" : "/var/lib/HPCCSystems/mydropzone/",
+        "dropzonePath" : "/home/jsmith/hpcc3Debug/var/lib/HPCCSystems/mydropzone/",
```

Comment:
I don't think this should have been checked in

Reply:
removed